src/service/xhr-impl.js

```diff
@@ -191,8 +193,8 @@ export class Xhr {
       init.headers['Content-Type'] = 'application/json;charset=utf-8';
```

Comment:
There's a test checking that this property is present in the request headers, and the second argument to setupInit sets the `Accepts` header, not `Content-Type`.
I can change the test to succeed, but I just want to make sure this is what you really intended.